packages/components/src/components/toast/toast/__snapshots__/toast-styles.spec.ts.snap

```diff
@@ -44,10 +44,10 @@ exports[`getComponentCss() should return correct css 1`] = `
 }
 
   .hydrated {
-    animation: 600ms in cubic-bezier(0.45,0,0.55,1) forwards;
+    animation: 0.4s in cubic-bezier(0,0,0.2,1) forwards;
   }
   .close {
-    animation: 600ms out cubic-bezier(0.5,1,0.89,1) forwards !important;
+    animation: 0.4s out cubic-bezier(0.4,0,0.5,1) forwards !important;
```

Comment:
Is this correct? Currently it appears there is a mismatch here with `MOTION_DURATION` which is 6000 or 0.6s?

Reply:
This change from 0.6s to 0.4s was in alignment with design